opus_codec_registration.src.html

```diff
@@ -131,16 +131,27 @@
 <xmp>
 dictionary OpusEncoderConfig {
   OpusBitstreamFormat format = "opus";
+  [EnforceRange] unsigned long frameDuration;
 };
 </xmp>
 </pre>
 
+To check if an {{OpusEncoderConfig}} is a valid OpusEncoderConfig,
```

Comment:
NIT: `<dfn>valid OpusEncoderConfig</dfn>`

Reply:
Cause there is not a more detail dfn about "valid OpusEncoderConfig", maybe a clever desc is:
`To check if an {{OpusEncoderConfig}} is valid,`
......